app/src/lib/tokens/PopupTable.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+	type T = $$Generic
```

Comment:
interesting, I've not seen this before... 🤔

Reply:
Yeah, had to do some Google searching. Otherwise `entries` would have to be `any` to accommodate different array types. Unfortunately the linter flags it as an error, but it works! I'm not super keen on using `lang="ts"` when we don't use it anywhere else, but it's the only way I found to make it do what I want.

Reply:
`lang="ts"` is no biggie.  It's not something we have to be consistent with across our components so feel free to use it when it helps.